manifests/stage-sriov-device-plugin/0020-sriov-dp-daemonset.yml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
```

Comment:
separate file ?

Reply:
Done. This was copy-pasted from Device Plugin documentation